### ISSUE TITLE
Vertical bars on line chart

### ISSUE THREAD

**Issue by gtnscebba** [FlowFuse/node-red-dashboard#1496]
### Description

I have several graphs on my dashboard and of different types; there are some, like the one shown, that highlight the temperatures for each hour of the day (from 00 to 23): as you can see, in the grid the vertical lines are every five hours and not every one.
Is there a way to force the vertical line every hour?

Since the grid is updated continuously (every hour the values ​​are updated), would it be possible to set a vertical line with a larger thickness (for example bold) to say "last update is this")? Of course, this line would be set by the program that creates the data.

In other words, the desired meaning is this: to the left of this bold bar the hours refer to the current day, and those after are from the previous day.

![image](https://github.com/user-attachments/assets/9c6e4d18-f4cb-446b-986f-05b4473fdf8c)

### Have you provided an initial effort estimate for this issue?

I'm not a member of FlowFuse

**Comment by gtnscebba** [comment]
Good morning,
maybe is better delete this request, correct?

**Comment by gtnscebba** [comment]
I immagine that is better to close this issue.
